src/io/export.h

```diff
@@ -101,7 +102,8 @@ bool is2D(const FileFormat format);
 bool exportFileByName(const std::shared_ptr<const class Geometry>& root_geom, const ExportInfo& exportInfo);
 
 void export_stl(const std::shared_ptr<const Geometry>& geom, std::ostream& output,
-                bool binary = true);
+                bool binary = true,
+                std::string solidName = "OpenSCAD Model");
```

Comment:
Do you have an idea of a better location? I figured with the `binary` default value being set here it would make sense to also put the `solidName` default value here as well.